### ISSUE TITLE
Noob issue: Can't import self-hosted Wordpress

### ISSUE THREAD

**Issue by NginUS** [jekyll/jekyll-import#220]
Hi,

I'm trying to import a Wordpress hosted on localhost with Xampp, using the instructions here [1], & I get this [2] error when I run wordpress.rb. I have tried both this [3] & this [4] in the wordpress.rb file.

I installed the jekyll-import gem, & am not sure what I'm doing wrong. I imagine its something way off, as I barely understand all these instructions I'm going through in the docs [5].

Would someone kindly ELI5 how to get this working, please?

Thanks,

-N

[1]
http://import.jekyllrb.com/docs/wordpress/

[2]
https://imgur.com/Qll5n3t

[3]
http://pastebin.com/ScH8Mfmy

[4]
http://pastebin.com/HdQTRyUw

[5]
http://jekyllrb.com/docs/migrations/

**Comment by parkr** [comment]
Based on your screenshot, it looks like you omitted the single quote (`'`) that wraps `require ... })`.
